examples/basic/05_error_handling.py

```
@@ -0,0 +1,272 @@
+
+    except Exception as e:
+        # OpenAI의 경우 AuthenticationError가 발생할 수 있음
+        if "authentication" in str(e).lower() or "api key" in str(e).lower():
+            print("❌ 인증 실패: API 키를 확인해주세요.")
+            print(f"   에러 메시지: {str(e)[:100]}...")
+        else:
+            print(f"❌ 예상치 못한 에러: {type(e).__name__}")
+            print(f"   에러 메시지: {str(e)[:100]}...")
+    finally:
+        # API 키 복원
+        os.environ["OPENAI_API_KEY"] = old_key
```

Pylint (3.3.7)

[error] Using variable 'old_key' before assignment

(E0601)